packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -502,6 +506,10 @@ contract StableToken is
    * @param tipTxFee Coinbase fee
    * @param baseTxFee Community fund fee
    * @param gatewayFee Gateway fee
+   * @dev Note that this function is called by the protocol when paying for tx fees in this
+   * currency. Before the tx is executed, gas is debited from the sender via a call to
+   * `debitGasFees`. Note too that the events emitted by `creditGasFees` reflect the *net* gas fee
+   * payments for the transaction.
```

Comment:
maybe mention that this must emit events?

I guess there's a difference between specifying the interface every ERC20 with gas fee support should implement, and what stabletoken does.. Of course should be the same, but from documenting perspective it's different (interface specifies what implementers should do)